lib/Discovery_Schema.cpp

```
@@ -0,0 +1,2188 @@
+
+	if (!resultset || resultset->rows_count == 0) {
+		proxy_warning("No run found for schema '%s'\n", run_id_or_schema.c_str());
+		if (resultset) {
+			free(resultset);
+			resultset = NULL;
+		}
+		return -1;
+	}
+
+	SQLite3_row* row = resultset->rows[0];
+	int run_id = atoi(row->fields[0]);
```

Cppcheck (2.19.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)